packages/nitro-protocol/hardhat-deployments/rinkeby/solcInputs/421853c27fa211830d511ef4302cc5f0.json

```diff
@@ -0,0 +1,50 @@
+    }
+  },
+  "settings": {
+    "optimizer": {
+      "enabled": false,
+      "runs": 200
+    },
+    "outputSelection": {
+      "*": {
+        "*": [
+          "abi",
+          "evm.bytecode",
+          "evm.deployedBytecode",
+          "evm.methodIdentifiers",
+          "metadata",
+          "devdoc",
+          "userdoc",
+          "storageLayout",
+          "evm.gasEstimates"
+        ],
+        "": [
+          "ast"
+        ]
+      }
+    },
+    "metadata": {
+      "useLiteralContent": true
+    }
+  }
+}
```

Comment:
Is this file meant to be ignored?

Reply:
I think we can probably ignore the `solcInputs` directory. f6dfcd5be